src/components/10-atoms/carousel/index.js

```diff
@@ -45,7 +50,7 @@ class AXACarousel extends InlineStyles {
     // Internal
     this.autoRotateTimerID = null;
     this.slides = null;
-    this.visibleSlideIndex = 0;
+    this.visible = 0;
```

Comment:
Why renamed it?

Reply:
Easy - because it is no externally visible as an attribute, and those tend to be short and to the point!